libvcx/src/v3/handlers/connection/mod.rs

```diff
@@ -29,6 +29,7 @@ pub mod tests {
         use v3::test::{Faber, Alice};
         use v3::messages::ack::tests::_ack;
         use v3::messages::a2a::A2AMessage;
+        use v3;
```

Comment:
Let's be consistent with importing and use `v3::messages::issuance::credential_offer::CredentialOffer`